models/confirmation.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"log"
+	mathRand "math/rand"
```

Comment:
I think it is preferable to use the crypto random generator for this:
https://golang.org/pkg/crypto/rand/

Reply:
that's correct.
let me psuh an update with crypto/rand